modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,50 +3,58 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can have multiple instances in the same environment. 
+In API Manager 2.x, instances of an API can exist in multiple environments. Additionally, you can create multiple instances of an API in the same environment. 
 
-You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
+An example: 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+* Create an instance of an API and configure it as a proxy. Apply caching policies to this proxy. 
 
-To create a API instance:
+* Then, create an API instance to manage directly as a basic endpoint. Apply throttling and security policies to this basic endpoint. 
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+* After configuring an API version for management, repeat the process to create another API instance. Specifying the same version creates another API instance of the version. 
+
+* Finally, create multiple API instances and click the version information to view API details.
+
+The following procedure describes creating an API instance from Exchange, and configuring the API instance as an endpoint with proxy.
+
+== Create an API instance
+
+. From *API Administration*, click *Manage API*. 
+. Select *Manage API From Exchange*.
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+To configure version 1.0 of an example API named `jsonplaceholder`:
 +
 image::manage-exchange-end-task.png[]
 +
-. From *API Name*, select the API (`jsonplaceholder`).
-+
-NOTE: OpenAPI Specification (OAS) 3.0 is currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager. For converting the specifications to OAS 2.0, you must use third-party tools, such as APImatic.
+. From *API name*, enter `jsonplaceholder`.
 +
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
-. From *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*. 
+NOTE: OpenAPI Specification (OAS) 3.0 is supported, with the exception of the callback feature. To work around this issue, handle the callback outside of the Mule runtime engine domain or use an OAS 3.0 specification that does not use callbacks.
```

Comment:
Try to avoid breaking up the steps of a task by inserting a notice.
Does this info belong here; is it specific to the step directly above it?
If not, move it outside of the procedure, either before or after, as most useful to the audience.

Reply:
```suggestion
Although OpenAPI Specification (OAS) 3.0 is supported, the callback feature is not. To work around this issue, either handle the callback outside of the Mule runtime engine domain or use an OAS 3.0 specification that does not use callbacks.
```

Reply:
Fixed. I added a "Notes" section at the bottom and moved this there.